packages/flutter_tools/test/general.shard/ios/xcodeproj_test.dart

```diff
@@ -641,10 +646,21 @@ Information about project "Runner":
   testWithoutContext('scheme for default project is Runner', () {
     final XcodeProjectInfo info = XcodeProjectInfo(<String>['Runner'], <String>['Debug', 'Release'], <String>['Runner'], logger);
 
-    expect(info.schemeFor(BuildInfo.debug), 'Runner');
-    expect(info.schemeFor(BuildInfo.profile), 'Runner');
-    expect(info.schemeFor(BuildInfo.release), 'Runner');
-    expect(info.schemeFor(const BuildInfo(BuildMode.debug, 'unknown', treeShakeIcons: false)), isNull);
+    expect(info.schemeFor(hostAppProjectName: 'Runner', flavor: BuildInfo.debug.flavor), 'Runner');
+    expect(info.schemeFor(hostAppProjectName: 'Runner', flavor: BuildInfo.profile.flavor), 'Runner');
+    expect(info.schemeFor(hostAppProjectName: 'Runner', flavor: BuildInfo.release.flavor), 'Runner');
```

Comment:
Again I think these are all the same because `BuildInfo.*.flavor` is null
```suggestion
    expect(info.schemeFor(hostAppProjectName: 'Runner'), 'Runner');
```